solidity/contracts/converter/interfaces/IConverter.sol

```diff
@@ -20,16 +19,13 @@ interface IConverter is IOwned {
                      address _trader,
                      address payable _beneficiary) external payable returns (uint256);
 
-    function conversionWhitelist() external view returns (IWhitelist);
     function conversionFee() external view returns (uint32);
     function maxConversionFee() external view returns (uint32);
-    function reserveBalance(IERC20Token _reserveToken) external view returns (uint256);
```

Comment:
this should be part of the interface even though it's not used internally by the contracts - only reason it's not is the backward compatibility